std_msgs/Bool capture
std_msgs/Bool restart
std_msgs/Bool quit
